cedar-policy/CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- Reduced precision of partial evaluation when evaluating lazy
+  expressions
```

Comment:
Probably makes more sense under the "changed" header than "removed" header. It also may be unclear to users what lazy expressions means in this context, so I'd suggest just
```suggestion
- Reduced precision of partial evaluation for `||`, `&&`,  and conditional expressions.
```